ceagle/main.py

```diff
@@ -13,61 +13,59 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-import json
+from __future__ import print_function
 import os
 
 import flask
 
-from ceagle.services.v1.capacity import capacity
-from ceagle.services.v1.cloud_status import cloud_status
-from ceagle.services.v1.infrastructure import infrastructure
-from ceagle.services.v1.intelligence import intelligence
-from ceagle.services.v1.optimization import optimization
-from ceagle.services.v1.security import security
+from ceagle.services import v1
 
 
 app = flask.Flask(__name__)
+api_map = []
 
-app.config.from_object(__name__)
-app.config.update({"SECRET_KEY": "change_this_key_in_prod"})
-app.config.from_envvar("CEAGLE_SETTINGS", silent=True)
 
+def load_config():
+    conf_defaults = {"SECRET_KEY": "change_this_key_in_prod"}
+    app.config.update(conf_defaults)
 
-@app.errorhandler(404)
-def not_found(error):
-    return flask.jsonify({"error": "Not found"}), 404
-
-
-for bp in [cloud_status, infrastructure, intelligence, optimization, security,
-           capacity]:
-    for url_prefix, blueprint in bp.get_blueprints():
-        app.register_blueprint(blueprint, url_prefix=url_prefix)
-
+    settings = os.environ.get("CEAGLE_CONF", "settings_local")
+    try:
+        app.config.from_object(settings)
+    except ImportError:
+        print("Module '%s' is not found. Using defaults..." % settings)
 
-CONFIG_LOADED = False
 
+def register_routes(svc_classes, uri_prefix=""):
```

Comment:
I would prefer to use just Flask bleuprints as routing mechanism, instead of implementing our own. Because it provides all functionality that we need.